packages/flutter/lib/src/material/icon_theme.dart

```diff
@@ -56,8 +56,13 @@ class IconTheme extends InheritedWidget {
   /// IconThemeData theme = IconTheme.of(context);
   /// ```
   static IconThemeData of(BuildContext context) {
-    IconTheme result = context.inheritFromWidgetOfExactType(IconTheme);
-    return result?.data ?? Theme.of(context).iconTheme;
+    IconThemeData themeData = _getInheritedIconThemData(context);
```

Comment:
can you call this variable iconThemeData? I briefly got confused and thought it was a ThemeData.